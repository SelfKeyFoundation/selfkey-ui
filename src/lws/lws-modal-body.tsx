import * as React from 'react';
import { WithStyles, withStyles, createStyles } from '@material-ui/core';

const styles = createStyles({
    root: {
        '& a': {
          color: '#23E6FE',
            textDecoration: 'none',
        },
        '& a &:hover': {
            textDecoration: 'underline'
        },
        padding: '60px 70px 80px',
    },
});

export type LWSModalBodyProps = {
    children?: any
};

export const LWSModalBody = withStyles(styles)(({ classes, children }: LWSModalBodyProps & WithStyles<typeof styles>) => (
    <div className={classes.root}>
        {children}
    </div>
));

export default LWSModalBody;
