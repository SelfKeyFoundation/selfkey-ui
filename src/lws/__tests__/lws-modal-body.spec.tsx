import * as React from 'react';
import { create } from 'react-test-renderer';
import { LWSModalBody } from '../lws-modal-body';

describe('LWS Modal Body', () => {
	it('should show LWS Modal Body', () => {
		const component = create(<LWSModalBody />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
