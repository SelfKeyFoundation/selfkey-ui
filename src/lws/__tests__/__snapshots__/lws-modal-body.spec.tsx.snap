// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LWS Modal Body should show LWS Modal Body 1`] = `
<div
  className="Component-root-1"
/>
`;
